cns/restserver/api_test.go

```diff
@@ -706,7 +707,7 @@ func createOrUpdateNetworkContainerWithParams(t *testing.T, params createOrUpdat
 	ipSubnet.IPAddress = params.ncIP
 	ipSubnet.PrefixLength = 24
 	ipConfig.IPSubnet = ipSubnet
-	podInfo := cns.KubernetesPodInfo{PodName: "testpod", PodNamespace: "testpodnamespace"}
```

Comment:
These are multitenant scenarios, we dont need these changes as marshalling KubernetesPodInfo should work too, can you revert these changes?